From 181bcc08dcdd7e029aabc524b777fda24feb9cea Mon Sep 17 00:00:00 2001
From: Daniel Hoffend <dh@dotlan.net>
Date: Mon, 18 Nov 2019 12:03:36 +0100
Subject: [PATCH 3/3] roundcubemail plugin owncload is no longer part of
 roundcubemail-plugins-kolab

Summary: there's no plugin ownload anymore, therefore the template for setup-kolab can safely get removed.

Reviewers: #pykolab_developers, #roundcube_kolab_plugins_developers

Differential Revision: https://git.kolab.org/D841
---
 share/templates/roundcubemail/owncloud.inc.php.tpl | 9 ---------
 1 file changed, 9 deletions(-)
 delete mode 100644 share/templates/roundcubemail/owncloud.inc.php.tpl

diff --git a/share/templates/roundcubemail/owncloud.inc.php.tpl b/share/templates/roundcubemail/owncloud.inc.php.tpl
deleted file mode 100644
index 513a482..0000000
--- a/share/templates/roundcubemail/owncloud.inc.php.tpl
+++ /dev/null
@@ -1,9 +0,0 @@
-<?php
-    // ownCloud URL
-    \$config['owncloud_url'] = 'http://' . \$_SERVER["HTTP_HOST"] . '/owncloud';
-
-    if (file_exists(RCUBE_CONFIG_DIR . '/' . \$_SERVER["HTTP_HOST"] . '/' . basename(__FILE__))) {
-        include_once(RCUBE_CONFIG_DIR . '/' . \$_SERVER["HTTP_HOST"] . '/' . basename(__FILE__));
-    }
-
-?>
-- 
2.20.1

